code/modules/vending/games.dm

```diff
@@ -25,6 +25,9 @@
 			"products" = list(
 				/obj/item/toy/captainsaid = 1,
 				/obj/item/toy/intento = 3,
+				// SKYRAT EDIT CHANGE - Adds laser pointers to the Good Clean Fun vendor
+				/obj/item/laser_pointer = 3,
```

Comment:
Not the way to do this. Look in the modular_vending file for the way we do this.